purchase.adoc

```diff
@@ -1,306 +1,461 @@
-## Purchase API
+= Purchase API
 
-### Service Description
-The purchase API provides information about purchases made through iZettle.
+== Description
+
+====
+Provides information about purchases made through iZettle.
+====
+
+=== Base URL
 
-### URL
 https://purchase.izettle.com
 
-### Scopes
-The Purchase API implements the following scopes:
+=== OAuth scopes
 
 - `READ:PURCHASE`
 
-=== Get purchase history
-Will include cash register information if applicable. Purchases up to three years old are available to fetch.
+'''
+
+==== Fetch a list of purchases
+
+Returns a list of purchases for a merchant.
+Purchases up to three years old are available to fetch.
 
 `GET /purchases/v2`
 
-lastPurchaseHash (optional):: A value from "lastPurchaseHash" from the result of a previous history query, to continue listing purchases from the next record after the previous query.
-startDate (optional):: The start date (inclusive) for purchases to be retrieved from until today or endDate. By default startDate is resolved to three years back.
-endDate (optional):: The last date (exclusive) for purchases to be retrieved until.
-limit (optional):: The maximum number of records to return. Max value of limit is 1000.
-descending (optional):: When true, returns purchases with the highest timestamp first.  When false, returns purchases with the lowest timestamp first. Defaults to false if not specified.
+[%collapsible]
+====
+*Parameters*
+
+[cols="15%,10%,10%,10%,55%"]
+|===
+|Name|Type|In|Required/Optional|Description
+
+|lastPurchaseHash|string|query|optional|A value from "lastPurchaseHash" from the result of a previous history query, to continue listing purchases from the next record after the previous query.
+|startDate|string|query|optional|The start date UTC(inclusive) for purchases to be retrieved from until today or endDate if provided e.g. "2020-01-01", "2020-01-01T17:00". +
+By default `startDate` is resolved to three years back.
+|endDate|string|query|optional|The last date UTC(exclusive) for purchases to be retrieved until e.g. "2021-01-13", "2021-01-13T12:30".
+|limit|integer|query|optional|The maximum number of records to return. Value range 1–1000.
+|descending|boolean|query|optional|When true, returns purchases with the highest timestamp first. When false, returns purchases with the lowest timestamp first. Defaults to false if not specified.
+|===
+
 
-Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation of the UUID fields.  The "UUID" fields are Base64 encoded UUID.
+*Responses*
 
-#### Permissions required
-`READ:PURCHASE`
+`Status: 200 OK` - Successful. +
+`Status: 401 Unauthorized` - You were not authorized to execute this operation. +
+`Status: 429 Too Many Requests` - You hit the rate limit. Retry request later.
 
-#### Example response 1
-`GET /purchases/v2?limit=10`
-```json
+*Examples*
+
+* Fetching limited number of purchases in descending order(most recent first).
```

Comment:
Sorry, Natalia, I forgot to mention this earlier. Could you also change "Fetching" to "Fetch"?